mail_optional_follower_notification/models/mail_thread.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.tools import config
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _notify_get_recipients(self, message, msg_vals, **kwargs):
+        """Compute recipients to notify based on subtype and followers. This
+        method returns data structured as expected for ``_notify_recipients``."""
+        test_condition = config["test_enable"] and not self.env.context.get(
```

Comment:
why do we need this test condition? :thinking:

Reply:
I didn't modify the existing approach.
It seems this was added to fix Travis build while migrating to v14.
https://github.com/OCA/social/commit/05c8aea71a38fcce24659f9d03e9653f383be677

Now, probably we can remove it as tests are green

Reply:
Well, there's no explanation of that change on that commit because it was squashed on the migration one :man_shrugging: